readme.md

```diff
@@ -701,6 +701,17 @@
 | ------: | -------: | ------: |
 |   1 - 4 |       14 | 60-120m |
 
+### [Ambush!](https://boardgamegeek.com/boardgame/1608/ambush)
```

Comment:
```suggestion
<!--lint ignore no-heading-punctuation-->
### [Ambush!](https://boardgamegeek.com/boardgame/1608/ambush)
```

This is a particular case in which the game is punctuated, so I'm overriding the linter's rule for this one.